packages/map/src/hooks/useGeoClickHandler.ts

```diff
@@ -1,9 +1,11 @@
 import ConfigContext from '../context'
 import { navigationHandler } from '../helpers'
 import { useContext } from 'react'
+import useMapDispatch from './useMapDispatch'
 
 const useGeoClickHandler = () => {
   const { state, setState, setModal, setSharedFilter, customNavigationHandler } = useContext(ConfigContext)
+  const dispatch = useMapDispatch()
```

Comment:
just call MapDispatchContext directly instead of creating this abstraction.

const dispatch = useContext(MapDispatchContext);